index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img
+          src="images/Dr. Norman.jpg"
+          alt="Dr. Norman Borlaug Picture"
+          id="image"
+        />
+        <h3 id="img-caption">Dr Norman in the field</h3>
+      </div>
+      <p id="tribute-info">Here's a time line of Dr. Borlaug's life:</p>
+      <a
+        href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+        id="tribute-link"
+        target="_blank"
+        >Read More.....</a
+      >
+    </div>
```

Comment:
Please use HTML 5 main tags for your web page main section.
I suggest you should also use class attributes over the id attribute

Reply:
- [ ] Kindly indicate why the developer needs to use the `<main>` tag.